src/utils/timeDateUtils.js

```diff
@@ -0,0 +1,32 @@
+import moment from 'moment';
+
+export function updateMomentLocale(lang) {
+  if (lang) {
+    switch (lang) {
+      case 'cn': {
+        moment.updateLocale('zh-cn', {
+          relativeTime: {
+            past: '%s 前',
```

Comment:
Can you add a note in the translations section of the `README` about this? We usually do translations manually, but since this is an easier way of doing it for this particular feature, I think we should just take note of this exception to the rule so future contributors won't get confused.